README.md

```diff
@@ -434,7 +455,104 @@ const sdk = await init();
 sdk.applicationNavigator.openContentItem({ id: 'CONTENT_ITEM_ID...' });
 ```
 
-# Client (Supported on Content Field & Dashboard extensions)
+# Content Editor
+
+### Get Current Model
+
+Fetching will give you the current model of the content item you're viewing.
+
+```js
+const sdk = await init();
+const model = await sdk.form.getValue();
+```
+
+### Set Model
+
+Setting the model will return Error Report if there is any invalid content
+
+```js
+const sdk = await init();
+try {
+  await sdk.form.setValue({ title: "hello world", })
+} catch (errors) {
+  // retuens error report 
+}
+
+if (errors.length) {
+  console.log(errors);
+}
+```
+
+### Validate
+
+You can validate an object against the schema and get the full error report back
+
+```js
+const sdk = await init();
+const errors = await sdk.form.validate({
+  title: 'hello world',
+});
+
+if (errors) {
+  console.log(errors);
+}
+
+//
+[
+  {
+    errorCategory: 'DATA',
```

Comment:
Maybe copy existing methods of showing example responses and use an Example title plus code block instead. Otherwise should be multiline comment